src/components/Icon/Icon.tsx

```diff
@@ -5,68 +5,149 @@ import svg4everybody from 'svg4everybody';
 import styles from './Icon.module.css';
 import icons from '../../icons/spritemap/spritemap.svg';
 
-export interface Props {
+interface IconPropsBase {
   /**
    * CSS class names that can be appended to the component.
    */
   className?: string;
   /**
-   * Declares whether the icon is focusable or not for accessibility
+   * Either a fragment of svg elements (g, path, circle, etc.), or a single
+   * svg element. Useful for creating specific icon components.
+   *
+   * @example
```

Comment:
we support this use case in the existing component, so figured it doesn't hurt to keep it